meta-lmp-base/recipes-security/optee/optee-os_git.bb

```diff
@@ -50,7 +34,7 @@ EXTRA_OEMAKE = "PLATFORM=${OPTEEMACHINE} O=out/arm \
                 CROSS_COMPILE_core=${HOST_PREFIX} \
                 CFG_WERROR=y DEBUG=0 LDFLAGS= \
                 LIBGCC_LOCATE_CFLAGS=--sysroot=${STAGING_DIR_HOST} \
-                CFG_TEE_CORE_LOG_LEVEL=2 CFG_TEE_TA_LOG_LEVEL=2 \
+                CFG_TEE_CORE_LOG_LEVEL=2 CFG_TEE_TA_LOG_LEVEL=2 CFG_CAAM_DBG=0x001 \
```

Comment:
This CFG_CAAM_DBG=0x001 setting is new.  Is that to limit the debug output or turn it on?

Reply:
to limit: is a bitmask (12 bits IIRC)  we could set it up to 0 but is good to remember we have this tweak IMO

Reply:
Ok

Reply:
Can we move this to an append specifically for imx-based devices?

Like 'EXTRA_OEMAKE_append_imx = " etc".

Reply:
@ricardosalveti yes, you are right. ok